app/src/app.js

```diff
@@ -67,6 +67,9 @@ app.use(require('koa-cash')({
     set(key, value) {
         logger.debug('Setting in cache. key: %s, value: ', key, value);
         cache.set(key, value);
+    },
+    hash() {
+        return this.request.originalUrl;
```

Comment:
could you clarify the motivation for this change?

Reply:
The cache key by default seemed to be based on the request.url, which removed the versioning in the URL. Consequently, calling the same API on different versions would hit the cache. I noticed this because when causes testing both versions at once, it picks up wrong responses from the cache. 

The original URL field keeps the version, so made more sense to me to cache on. Let me know if there's a more standard way to deal with this issue.

Reply:
oh, that's a great finding. I never used koa-cash myself, but I think I've seen it used in other microservices, and I'm not sure this problem was taken into account. I'll keep this in mind for the future. Thanks!